prep-javascript-arrays/main.js

```diff
@@ -0,0 +1,19 @@
+const languages = ['html', 'css', 'javascript', 'react', 'node', 'postgres'];
+
+console.log(languages);
+
+const firstElement = languages[0];
+
+console.log('the first element of the array is', firstElement);
+
+const thirdElement = languages[2];
+
+console.log('the 3rd element of the array is', thirdElement);
+
+const length = languages.length;
+
+console.log('The length of the array is', length);
+
+const lastElement = languages[5];
```

Comment:
Rather than using `5` directly, you should be subtracting 1 from the `length` property of  `languages`

Reply:
My bad, I had written that in my quiz answers, but forgot to use it in the actual code. Thanks!